src/Symfony/Bridge/Twig/AppVariable.php

```diff
@@ -49,6 +52,17 @@ public function setDebug(bool $debug)
         $this->debug = $debug;
     }
 
+    public function setTwig(Environment $twig)
```

Comment:
Do we need the setter? Why not use constructor injection instead?

Reply:
Method removed